.github/workflows/ci-aarchxx.yml

```diff
@@ -55,6 +55,8 @@ jobs:
 
```

Comment:
The PR title:

> i#5517 add submodule libipt: Add libipt & Initialize submodules all the time #5518

The convention is to only use the short issue description before the colon for issue numbers addresses with multiple PR's.  Since you've made this single issue just for this submodule, with presumably a single PR addressing it, we would expect to not have that before-colon issue description.  (Or, put this PR under the #5505 issue and close #5517 as a duplicate/part of #5505.  If not doing that: please add a ref to #5505 in the Issue: list in the commit description.)

Reply:
Done.